alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,91 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator
+rejects invalid `AlertmanagerConfig`, `PodMonitor`, `ServiceMonitor`, or `PrometheusRules`
+objects.
+
+## Impact
+
+The custom resources that trigger the alert are ignored by Prometheus Operator.
+As a consequence, they will not be part of the final configuration of the
+Prometheus, Alertmanager, or Thanos Ruler components managed by Prometheus Operator.
+
+## Diagnosis
+
+Find the custom resource type and the namespace in the `description` field
+of the `PrometheusOperatorRejectedResources` alert. You can find this information
+by using the OpenShift web console or the command line interface (CLI).
+
+### Diagnose using the Openshift web console
+
+1. Browse to **Observe** -> **Alerting**.
+2. If the **Filter** for **Alert State** is not set to **Firing**,
+ click the **Filter** drop-down list and select **Firing**.
+3. Search for the `PrometheusOperatorRejectedResources` alert.
+4. Click the alert to view its details.
+5. Scroll down and view the **Description** field. The namespace is displayed as
+a link, followed by the resource type.
+
+After you know the namespace and the resource type, you can search for a recently
+modified custom resource of that type in that namespace.
```

Comment:
This paragraph is a bit unclear to me. The namespace label can be either openshift-monitoring or openshift-user-workload-monitoring but the faulty resource might live elsewhere.

Reply:
Maybe add the following sentence: `

`However, note that the namespace label in the description can be either `openshift-monitoring` or `openshift-user-workload-monitoring`, but the faulty resource might still be located elsewhere.`